css/syntaxes.json

```diff
@@ -254,6 +284,9 @@
   "grid-line": {
     "syntax": "auto | <custom-ident> | [ <integer> && <custom-ident>? ] | [ span && [ <integer> || <custom-ident> ] ]"
   },
+  "hash-token": {
```

Comment:
`hash-token` is a token type and out of scope of this dict.
Btw, it has another syntax